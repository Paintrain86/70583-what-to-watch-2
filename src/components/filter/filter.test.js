import React from 'react';
import renderer from 'react-test-renderer';
import Filter from './filter.jsx';

const filters = [
  {
    genre: `crime`,
    name: `Crime`
  },
  {
    genre: `documentary`,
    name: `Documentary`
  },
  {
    genre: `drama`,
    name: `Dramas`
  }
];

describe(`Filter`, () => {
  it(`component should render correctly`, () => {
    const markup = renderer
      .create(<Filter filterItems={filters}/>)
      .toJSON();

    expect(markup).toMatchSnapshot();
  });
});
