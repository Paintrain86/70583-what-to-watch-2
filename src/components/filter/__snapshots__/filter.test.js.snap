// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Filter Filter should render correctly 1`] = `
<ul
  className="catalog__genres-list"
>
  <li
    className="catalog__genres-item"
  >
    <a
      className="catalog__genres-link"
      href="#"
      onClick={[Function]}
    >
      Crime
    </a>
  </li>
  <li
    className="catalog__genres-item"
  >
    <a
      className="catalog__genres-link"
      href="#"
      onClick={[Function]}
    >
      Documentary
    </a>
  </li>
  <li
    className="catalog__genres-item"
  >
    <a
      className="catalog__genres-link"
      href="#"
      onClick={[Function]}
    >
      Dramas
    </a>
  </li>
</ul>
`;
